backend/kernelCI_app/management/commands/notifications.py

```diff
@@ -482,7 +487,65 @@ def discard_sent_reports(
     return key_set_copy, prop_map_copy
 
 
+def get_build_issues_from_checkout(
+    *, checkout_ids: list[str]
+) -> tuple[dict[str, list[CheckoutIssue]], dict[str, list[str]]]:
+    checkout_issue_record = get_issues_summary_data(checkout_ids=checkout_ids)
+
+    checkout_issues: dict[str, list[dict]] = defaultdict(list)
+    build_without_issues: dict[str, list[str]] = defaultdict(list)
```

Comment:
It would make the variable name longer but I think it would be more specific:
```suggestion
    checkout_builds_without_issues: dict[str, list[str]] = defaultdict(list)
```